bee-test/src/rand/bytes.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::Rng;
+
+/// Generates a vector of random bytes with a given length.
+pub fn rand_bytes(len: usize) -> Vec<u8> {
+    (0..len).map(|_| rand::random::<u8>()).collect()
+}
+
+/// Generates an array of random bytes of length 32.
+pub fn rand_bytes_32() -> [u8; 32] {
+    rand::thread_rng().gen::<[u8; 32]>()
```

Comment:
Since 1.51, `rand` introduced the `min_const_gen` feature to allow being generic over arrays of more then 32 items. Maybe we can start using it and have a `rand_bytes_array` instead of simply `rand_bytes_32` ?

Reply:
https://docs.rs/rand/0.8.4/rand/trait.Rng.html#arrays-and-tuples

Reply:
Ah, nice!